### PR TITLE
DOC-2167: Clarify PHP versions for SDK

### PR DESCRIPTION
**Pull request by mattcarabine**

### CODE REVIEW

**Comment by mattcarabine** [comment]
@cb-docs-robot backport to 4.6

**Comment by mattcarabine** [comment]
@cb-docs-robot backport to 4.5

**Comment by mattcarabine** [comment]
@cb-docs-robot backport to 4.6

File: content/sdk/php/start-using-sdk.dita [hunk 1]
```diff
@@ -4,8 +4,9 @@
     <title>Install and Start Using the PHP SDK with Couchbase Server</title>
     <titlealts><navtitle>Start Using the SDK</navtitle></titlealts>
     <shortdesc>The Couchbase PHP SDK allows you to connect to a Couchbase cluster from PHP. The
-        Couchbase PHP SDK is a native PHP extension and uses the Couchbase high-performance C library
-        to handle communicating to the cluster over Couchbase binary protocols.</shortdesc>
+        Couchbase PHP SDK is a native PHP extension and uses the Couchbase high-performance C
+        library to handle communicating to the cluster over Couchbase binary protocols. The
+        Couchbase PHP SDK is compatible with both PHP 5.6 and 7.0.</shortdesc>
```
Comment (avsej) [inline content/sdk/php/start-using-sdk.dita]: @mattcarabine and also 5.4, 5.5 and 7.1

http://sdkbuilds.sc.couchbase.com/view/PHP/job/php-sdk-build-linux/
image

**Pull request merged**